sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -245,7 +245,7 @@ case class InsertIntoHiveTable(
             if (fs.exists(partitionPath)) {
               if (!fs.delete(partitionPath, true)) {
                 throw new RuntimeException(
-                  "Cannot remove partition directory '" + partitionPath.toString)
+                  "Cannot remove partition directory '" + partitionPath.toString + "'")
```

Comment:
We can even use a string interpolation: `s"Cannot remove partition directory '$partitionPath'"`

Reply:
seems the jvm performance the same, but string interpolation is shorter.